cmd/loop/loopout.go

```diff
@@ -75,8 +82,10 @@ func loopOut(ctx *cli.Context) error {
 	}
 	defer cleanup()
 
+	sweepConfTarget := int32(ctx.Uint64("conf-target"))
 	quoteReq := &looprpc.QuoteRequest{
-		Amt: int64(amt),
+		Amt:        int64(amt),
+		ConfTarget: sweepConfTarget,
```

Comment:
We should likely modify this command to be: `loop out quote` or `loop quote out` for symmetric consistency. Also the case that right now, there's no way to get a quote for a loop in quote on the command line.

Reply:
Seems like a good candidate for a follow-up PR?

Reply:
Yeah